content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -160,7 +160,11 @@ Users today need to use some combination of taints and node selectors in order t
 keep Linux and Windows workloads on their respective OS-specific nodes. 
 This likely imposes a burden only on Windows users. The recommended approach is outlined below, 
 with one of its main goals being that this approach should not break compatibility for existing Linux workloads.
-
+ {{< note >}}
+ If the `IdentifyPodOS` featuregate is enabled and the `pod.spec.os.name` field even if set to `windows` will have
+ no effect on the scheduling of the Windows pods, so we should still rely on taints and tolerations and node selector
```

Comment:
```suggestion
 no effect on the scheduling of the Windows pods, so taints and tolerations and node selectors are still required.
```

Maybe link to that section?

Reply:
I understand we can't point to the future in the docs but for my own understanding, once we are in Beta we should be able to use this field for scheduling?

Reply:
We've not decided on it yet. The main reason for not going in that direction atm is LCOW(where the nodeSelector may point to Windows but the podOS can be linux).